test/e2e/lib/install.bash

```diff
@@ -7,16 +7,16 @@ source "${E2E_DIR}/lib/template.bash"
 
 function install_tiller() {
   if ! helm2 version > /dev/null 2>&1; then # only if helm isn't already installed
-    kubectl --namespace kube-system create sa tiller
-    kubectl create clusterrolebinding tiller-cluster-rule --clusterrole=cluster-admin --serviceaccount=kube-system:tiller
-    helm2 init --service-account tiller --upgrade --wait
+    kubectl --namespace "$E2E_NAMESPACE" create sa tiller
+    kubectl create clusterrolebinding tiller-cluster-rule --clusterrole=cluster-admin --serviceaccount="$E2E_NAMESPACE":tiller
+    helm2 init --tiller-namespace "$E2E_NAMESPACE" --service-account tiller --upgrade --wait
   fi
 }
 
 function uninstall_tiller() {
-  helm2 reset --force
+  helm2 reset --tiller-namespace "$E2E_NAMESPACE" --force
   kubectl delete clusterrolebinding tiller-cluster-rule
-  kubectl --namespace kube-system delete sa tiller
+  kubectl --namespace "$E2E_NAMESPACE" delete sa tiller
```

Comment:
I guess this isn't really needed as we nuke the namespace but it does not hurt.

Reply:
This function is not aware of how it is utilized and should simply undo what its `install` companion creates, so this is perfect.